hypothesis-python/src/hypothesis/core.py

```diff
@@ -387,6 +387,24 @@ def skip_exceptions_to_reraise():
 exceptions_to_reraise = skip_exceptions_to_reraise()
 
 
+def failure_exceptions_to_catch():
+    """Return a tuple of exceptions meaning 'this test has failed', to catch.
+
+    This is intended to cover most common test runners; if you would
+    like another to be added please open an issue or pull request.
+    """
+    exceptions = [Exception]
+    try:  # pragma: no cover
```

Comment:
nit: It doesn't really matter, but I find it weird having the no cover on the `try` instead of the `except ImportError`

Reply:
Just matching the style of the blocks one function up :confused: